game_stats_fixtures/teams_fixtures.csv

```diff
@@ -1,8 +1,8 @@
-team_id,franchiseId,teamName,abbreviation,Stadium,link
-1,23,Atlanta United,ATL,Mercedes-Benz Stadium,/api/v1/teams/1
-4,16,Chicago Fire,CHI,SeatGeek Stadium,/api/v1/teams/4
-26,14,FC Cincinnati,CIN,Nippert Stadium,/api/v1/teams/26
-14,31,DC United,DC,Audi Field,/api/v1/teams/14
-6,6,FC Dallas,DAL,Toyota Stadium,/api/v1/teams/6
-3,10,Houston Dynamo,HOU,BBVA Stadium,/api/v1/teams/3
-5,17,Sporting Kansas City,SKC,Children's Mercy Park,/api/v1/teams/5
+team_id,franchiseId,teamName,abbreviation,Stadium,link
```

Comment:
@ajtran303 I went through and added all of your teams_fixtures from your branch so it should match your file exactly